src/PIL/PcfFontFile.py

```diff
@@ -55,7 +55,9 @@ class PcfFontFile(FontFile.FontFile):
 
     name = "name"
 
-    def __init__(self, fp):
+    def __init__(self, fp, charset_encoding='iso8859-1'):
```

Comment:
```suggestion
    def __init__(self, fp, charset_encoding="iso8859-1"):
```
Change to allow lint to pass.

Reply:
Done